css/properties/offset-path.json

```
@@ -233,7 +233,7 @@
         },
         "url": {
           "__compat": {
-            "description": "<code>&lt;url()&gt;</code>",
+            "description": "Support for <code>&lt;url&gt;</code>",
```

Given everything is about support in BCD, we usually don't add wording like this.

```suggestion
            "description": "<code>&lt;url&gt;</code>",
```